### PR TITLE
Make sure users know where to find the full docker picture.

### PR DESCRIPTION
**Pull request by chouseknecht**
##### ISSUE TYPE
- Docs Pull Request
##### COMPONENT NAME

docker.py
##### ANSIBLE VERSION

```
ansible 2.2.0 (doc-updates 22de8a9a21) last updated 2016/05/18 14:31:59 (GMT -400)
  lib/ansible/modules/core: (detached HEAD 92bf802cb8) last updated 2016/05/18 14:11:02 (GMT -400)
  lib/ansible/modules/extras: (detached HEAD e710dc47fe) last updated 2016/05/18 06:08:06 (GMT -400)
  config file = /etc/ansible/ansible.cfg
  configured module search path = Default w/o overrides
```
##### SUMMARY

Adding a link to Getting Started with Docker guide.

### CODE REVIEW

**Comment by willthames** [comment]
This seems to break tests (see #3698) - do we know why tests didn't run?

File: cloud/docker/docker.py [hunk 1]
```diff
@@ -27,7 +27,9 @@
 version_added: "1.4"
 short_description: manage docker containers
 description:
-  - Manage the life cycle of docker containers.
+  - This is the original Ansible module for managing the Docker container life cycle.
+  - NOTE: Additional and newer modules are available. For the latest on orchestrating containers with Ansible
+    visit our Getting Started with Docker Guide at U(https://github.com/ansible/ansible/blob/devel/docsite/rst/guide_docker.rst).
```
Comment (docschick) [inline cloud/docker/docker.py:32]: There's an extra character, a "U", before the URL.  The URL doesn't need parenthesis around it when written this way.

Comment (abadger) [inline cloud/docker/docker.py:32]: The U() should be parsed by our module documentation formatter code... it's part of the multiple parsers thing.   First python parses it as a string.  Then we load the docstring using yaml.  Then we find certain constructs like U(link)  and C(code_formatting) and substitute rst formatting for them. Finally, we hand it off to sphinx to format the rst into html.

Comment (docschick) [inline cloud/docker/docker.py:32]: I have to much to learn about how the module docs are built. Thanks @abadger!  And, @chouseknecht please ignore my above comment and apologies for not knowing this was correct.

**Pull request merged**